irohad/main/server_runner.cpp

```diff
@@ -5,15 +5,22 @@
 
 #include "main/server_runner.hpp"
 
+#include <fstream>
```

Comment:
According to the Google c++ style guide any STL includes have to be divided from the others with a blank line.
Please add a blank line.

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Fixed